dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/util/AnalyticsUtils.java

```diff
@@ -222,7 +222,7 @@ else if ( decimals != null && decimals > 0 )
         }
         else
         {
-            return MathUtils.getRounded( value );
+            return new Integer( (int) Precision.round( value, 0 ) );
```

Comment:
This will not work exactly as intended. We do have a "default rounding"  which applies when there is no rounding specified in the request which is implemented in MathUtils.getRounded. We need a special clause for when 0 decimals is explicitly specified.